keras/layers/wrappers.py

```diff
@@ -417,6 +434,48 @@ def call(self, inputs, training=None, mask=None, initial_state=None):
             return [output] + states
         return output
 
+    def _standardize_args(self, inputs, initial_state, constants):
+        """Standardize `__call__` to a single list of tensor inputs.
+
+        This method is taken from `RNN` as it is.
```

Comment:
Could you please split this method to a standalone utility function in `recurrent.py` (named `def _standardize_args`)? The only instance attribute it needs is `_num_constants`, which can be passed as a function argument (`num_constants`).

That way we don't need to duplicate this code, you can just import it from `recurrent.py`.